ambry-coordinator/src/test/java/com.github.ambry.coordinator/CoordinatorTest.java

```diff
@@ -1034,13 +1034,13 @@ public void simpleOneDCFourNodeOneDiskTwoPartition()
     simple(getClusterMapOneDCFourNodeOneDiskTwoPartition());
   }
 
-  @Test
```

Comment:
We can, it is just that this is going to be a one off thing then, there are several more important issues that we should probably move first?